website/.spelling

```diff
@@ -2070,6 +2070,7 @@ druid.enableTlsPort
 druid.indexer.autoscale.workerVersion
 druid.service
 druid.storage.disableAcl
+druid.labels
```

Comment:
This shouldn't be needed if we quote the property name as `druid.labels` in the docs.